src/libraries/System.CodeDom/tests/System/CodeDom/Compiler/CodeCompilerTests.cs

```diff
@@ -55,6 +55,7 @@ public void CompileAssemblyFromDom_NullOptions_ThrowsArgumentNullException()
 
         [Theory]
         [MemberData(nameof(CodeCompileUnit_TestData))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/73721", typeof(PlatformDetection), nameof(PlatformDetection.IsNodeJS), nameof(PlatformDetection.IsWindows))]
```

Comment:
This won't work, the platform is Browser not Windows

Reply:
Yea, anyway I'll try to make a working version using https://nodejs.org/api/process.html#process_process_platform

Reply:
Use platformdetection.IsBrowserOnWindows